### PR TITLE
Spontaneous combustion now has a minimal stack of 1 it applies

### PR DESCRIPTION
**Pull request by farie82**
**What does this PR do:**
Gives spontaneous combustion a minimum stack size of 1. Preventing the fire virus to make you wet instead.
fixes: #11152

**Changelog:**
:cl:
tweak: Spontaneous combustion now has a minimum stacksize of 1. No more getting wet from the fire virus
/:cl:

### CODE REVIEW

File: code/datums/diseases/advance/symptoms/fire.dm [hunk 1]
```diff
@@ -45,13 +45,13 @@ Bonus
 	return
 
 /datum/symptom/fire/proc/Firestacks_stage_4(mob/living/M, datum/disease/advance/A)
-	var/get_stacks = (sqrtor0(20+A.totalStageSpeed()*2))-(sqrtor0(16+A.totalStealth()))
+	var/get_stacks = max((sqrtor0(20 + A.totalStageSpeed() * 2)) - (sqrtor0(16 + A.totalStealth())), 1)
 	M.adjust_fire_stacks(get_stacks)
-	M.adjustFireLoss(get_stacks/2)
+	M.adjustFireLoss(get_stacks / 2)
```
Comment (Fox-McCloud) [inline code/datums/diseases/advance/symptoms/fire.dm]: `* 0.5`

Comment (farie82) [reply code/datums/diseases/advance/symptoms/fire.dm]: can also do >> 1. But *.5 is gut enough

Comment (Fox-McCloud) [reply code/datums/diseases/advance/symptoms/fire.dm]: the point is purely code related.

Multiplication is faster than division, so, where possible, utilize it (not always possible, I realize).

Comment (farie82) [reply code/datums/diseases/advance/symptoms/fire.dm]: Aye multiplication is faster. bitwise should be faster as well but I'm not sure if it'll work in DM due to it using floats/doubles

**Pull request merged**